src/main/java/com/codejoust/main/model/report/CodeLanguage.java

```diff
@@ -0,0 +1,32 @@
+package com.codejoust.main.model.report;
+
+import com.codejoust.main.exception.GameError;
+import com.codejoust.main.exception.api.ApiException;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import lombok.Getter;
+
+@Getter
+public enum CodeLanguage {
+
+    PYTHON,
```

Comment:
Any reason for duplicating this file? We could move the other one into a higher up directory that way it's more suited for reuse

Reply:
That's a good point - I removed the duplicate file and moved the old one to the `main/model/report` so that it's placed in the database.

Reply:
No reason, good point - I deleted this file and moved the original to the `main/model/report` folder.